src/Symfony/Component/Form/Tests/AbstractBootstrap4LayoutTest.php

```diff
@@ -940,9 +940,20 @@ public function testFile()
     {
         $form = $this->factory->createNamed('name', FileType::class);
 
-        $this->assertWidgetMatchesXpath($form->createView(), array('attr' => array('class' => 'my&class form-control-file')),
-'/input
-    [@type="file"]
+        $this->assertWidgetMatchesXpath($form->createView(), array('id' => 'nope', 'attr' => array('class' => 'my&class form-control-file')),
```

Comment:
I know that this `id`-thing here is weird, but the check doesn't make sense for this file type (because of the special structure), I guess.